base/doc/ltnews39.tex

```diff
@@ -725,6 +725,19 @@ \subsection{Struts at the end of footnotes or \texttt{p} columns}
 \pkg{bigfoot})}
 
 
+\subsection{Fix a ``missing \cs{item}'' rollback error}
+
+If \LaTeX{} is rolled back to a date not newer than 2023/06/01 and
+older than 2024/06/01, any list-based environment would raise an error
```

Comment:
not newer 2023 but older than 2024/06 ? I think you meant something like

rolled back to a date between ...

maybe?

Reply:
between also was not is as it's used with would so

If \LaTeX{} was rolled back to a date between  2023/06/01 and
2024/06/01, any list-based environment would raise an error

Reply:
The original "not newer than 2023/06/01" should be "not older than ...".

To express the constraint "2023/06/01 <= date < 2024/06/01", how about

```suggestion
If \LaTeX{} is rolled back to a date between 2023/06/01 (inclusive) and
2024/06/01 (exclusive), any list-based environment would raise an error
```

Reply:
looks fine